bitnami/schema-registry/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+appVersion: 6.0.1
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
+  - condition: kafka.enabled
+    name: kafka
+    repository: https://charts.bitnami.com/bitnami
+    version: 18.x.x
+description: Confluent Schema Registry provides a RESTful interface by adding a serving layer for your metadata on top of Kafka. It expands Kafka enabling support for Apache Avro, JSON, and Protobuf schemas.
+engine: gotpl
+home: https://confluent.io/
+icon: https://bitnami.com/assets/stacks/schema-registry-stac/img/schema-registry-stack-220x234.png
+keywords:
+  - schema-registry
+  - confluent
+  - kafka
+  - zookeeper
+  - streaming
+maintainers:
+  - email: containers@bitnami.com
```

Comment:
Is that correct?

Reply:
Good catch, it was copied from https://github.com/bitnami/commercial-charts that's why I assumed everything was fine since it is not a new asset, we are just moving it. But it seems that `commercial-charts` repo is not up-to-date